### PR TITLE
Improve grammar

### PR DESCRIPTION
**Pull request by ShadySuperCoder**
Improved grammar -- no code changes.

### CODE REVIEW

File: doc/dnsseed-policy.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ functioning Bitcoin nodes from the public network to the best of the
 operator's understanding and capability.
 
 2. For the avoidance of doubt, the results may be randomized but must not
-single-out any group of hosts to receive different results unless due to an
+single-out any group of hosts to receive different results, unless due to an
```
Comment (Zero-1729) [inline doc/dnsseed-policy.md:23]: Not sure about this punctuation here.

**Review by Zero-1729 — COMMENTED** [review]
Light code review and concept ACK.

Consider renaming your commit and the PR title to maybe `docs: minor grammar fixes to dnsseed-policy.md`, or something more specific and in accordance with the [Contributing](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#creating-the-pull-request) docs.

**Note**: I recommend going through the [contributing docs](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md).

**Comment by fanquake** [comment]
> ShadySuperCoder

> no code changes.

Thanks. However if you're looking to start contributing, I'd suggest looking at the [good first issue list](https://github.com/bitcoin/bitcoin/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). As mentioned, please also read our contributing docs. If you are going to open PRs like this, at least accompany them with a proper commit message.

**Comment by MarcoFalke** [comment]
Some more pointers:

## Getting started to contribute to Bitcoin Core

### Setting up your development environment

New developers are very welcome and needed. There are a lot of open issues of any difficulty waiting to be fixed. However, before you start contributing, familiarize yourself with the Bitcoin Core build system and tests. Refer to the documentation in the repository on how to build Bitcoin Core and how to run the unit and functional tests. Once that is done, you are all set.

If you need more help getting started, please refer to the following resources:

* https://medium.com/@amitiu/onboarding-to-bitcoin-core-7c1a83b20365
* https://jonatack.github.io/articles/how-to-review-pull-requests-in-bitcoin-core

### Pick something to work on

If you are looking for useful contributions to help out with, you can

* Search through the [good first issue](https://github.com/bitcoin/bitcoin/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)s or the ones that are [up for grabs](https://github.com/bitcoin/bitcoin/issues?utf8=%E2%9C%93&q=label%3A%22Up+for+grabs%22). Some of them might no longer be applicable. So if you are interested, but unsure, you might want to leave a comment on the issue first.
* Write tests to improve the coverage. Any kind of test is welcome and coverage information can be obtained from a relatively recent [coverage report](https://marcofalke.github.io/btc_cov). If you are unsure, don't hesitate to check back first.
* Help with review and testing. There are easy ones such as the [gui](https://github.com/bitcoin/bitcoin/pulls?q=is%3Apr+is%3Aopen+label%3AGUI) and [rpc](https://github.com/bitcoin/bitcoin/pulls?q=is%3Apr+is%3Aopen+label%3ARPC%2FREST%2FZMQ). However, review on any open pull request is welcome. Review will also help you understand the codebase better.
* Help on meta projects related to Bitcoin Core, such as a [high-level performance monitor](https://github.com/chaincodelabs/bitcoinperf).
* Join us on irc and let us know what you are interested in.

**Pull request closed without merging**